networks/remote/terraform/cluster/variables.tf

```diff
@@ -5,7 +5,7 @@ variable "name" {
 variable "regions" {
   description = "Regions to launch in"
   type = "list"
-  default = ["AMS2", "FRA1", "LON1", "NYC3", "SFO2", "SGP1", "TOR1"]
+  default = ["TOR1"]
```

Comment:
why this change?

Reply:
persistently get an error along the lines of "region not available" but terraform doesn't say which region. I know TOR1 works, so went with that rather than waste time tracking down which region is bunk.

Reply:
ok. but we still probably want 4 nodes in 4 different AZ

Reply:
ah, the culprit is `AMS2`, which should be `AMS3`